### PR TITLE
Update dependencies to fix some errors found during ESS

### PR DESCRIPTION
**Pull request by guioliveirabh**
### Fixes

- [EXDRHUB-1257](https://issues.redhat.com/browse/EXDRHUB-1257)

### Checklist

- [x] Related tests were updated
- [x] Run `tox`, no tests failed

### CODE REVIEW

File: sonar-project.properties [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# The projectKey will need to be updated once we have this on CI
+sonar.projectKey=guolivei-rhub-api
```
Comment (brhatiga) [inline sonar-project.properties:2]: Not sure that a specific keyname should be included here, probably should be anonymized

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+sonar.host.url=https://sonarqube.corp.redhat.com
```
Comment (brhatiga) [inline sonar-project.properties:3]: Same for the url

File: sonar-project.properties (hunk 1 above)
Comment (guioliveirabh) [reply sonar-project.properties]: Fixed on: https://github.com/resource-hub-dev/rhub-api/pull/147/commits/602b1cfaf6649838a04b4d8aadb111a4d6a1c283

File: sonar-project.properties (hunk 2 above)
Comment (guioliveirabh) [reply sonar-project.properties]: Fixed on: https://github.com/resource-hub-dev/rhub-api/pull/147/commits/602b1cfaf6649838a04b4d8aadb111a4d6a1c283

**Review by brhatiga — APPROVED** [review]

**Pull request merged**